### PR TITLE
Inconsistent error message with vm in an unknown state

### PR DESCRIPTION
**Pull request by roy9495**
Issue #3659
Earlier when vm was in unknown state it was providing different error messages between 'restart' and 'start'
Earlier
~~~
$ multipass restart charm-dev
restart failed: instance "charm-dev" is not running

$ multipass start charm-dev
start failed: The following errors occurred:
Instance 'charm-dev' is already running, but in an unknown state
~~~
Now
~~~

$ multipass restart charm-dev
restart failed: Instance 'Charm-dev' is already running, but in an unknown state
Use `multipass stop charm-dev`

$ multipass start charm-dev
start failed: The following errors occurred:
Instance 'Charm-dev' is already running, but in an unknown state
Use `multipass stop charm-dev`
~~~

### CODE REVIEW

**Review by ricab — COMMENTED** [review]
Thank you @roy9495! A couple tweaks and we should be good to go :slightly_smiling_face:

**Comment by ricab** [comment]
Hey @roy9495, we'd need you to apply [these formatting changes](https://github.com/canonical/multipass/actions/runs/10825539787/job/30044375030?pr=3670#step:3:90) and also [sign the CLA](https://ubuntu.com/legal/contributors).

File: src/daemon/daemon.cpp [hunk 1]
```diff
@@ -3110,7 +3112,10 @@ grpc::Status mp::Daemon::reboot_vm(VirtualMachine& vm)
 
     if (!MP_UTILS.is_running(vm.current_state()))
         return grpc::Status{grpc::StatusCode::INVALID_ARGUMENT,
-                            fmt::format("instance \"{}\" is not running", vm.vm_name), ""};
+                            fmt::format("Instance \"{}\" is already running, but in an unknown state\n"
```
Comment (ricab) [inline src/daemon/daemon.cpp:3115]: idem

**Comment by georgeliao** [comment]
@roy9495 
Thanks for the contribution. 
It looks good to me, the Positional Formatting usage is also fine here.

**Review by georgeliao — APPROVED** [review]

**Review by ricab — APPROVED** [review]

**Comment by ricab** [comment]
Hi @roy9495, this was merged in #3706. Thank you again!

**Comment by roy9495** [comment]
Thank you

**Pull request closed without merging**